test/integration/jobset_controller_test.go

```diff
@@ -113,8 +130,19 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 				}
 			}
 		},
+		// TODO: move validation tests to separate Describe + DescribeTable
+		ginkgo.Entry("jobset validation should fail if DNS hostnames is enabled and job completion mode is not indexed", &testCase{
```

Comment:
"validate enableDNSHostnames can't be set if job is not Indexed"

Reply:
Done